draft-ietf-dprive-dnsoquic.md

```diff
@@ -719,10 +726,8 @@ connections. For example:
 * DoQ implementations SHOULD
      * use the same QUIC connection for both AXFR and IXFR requests to the same
        primary
-     * pipeline such requests (if they pipeline XFR requests in general) and
-       MAY intermingle them
```

Comment:
I think we need to retain this requirement in some way - the point is that DoQ clients doing XFR MUST NOT wait for one XFR response before sending the next one (which several TCP implementations do today). Now this should be natural in QUIC but this bullet was supposed to spell that out:

suggestion:
"
* send those requests in parallel as soon as they are queued i.e. do not wait for a response before sending the next query on the connection (this is analogous to pipelining requests on a TCP/TLS connection)
"

Reply:
Yes. I edited the document accordingly.